assets/src/components/routePicker.tsx

```diff
@@ -17,11 +17,16 @@ interface Props {
 
 const RoutePicker = ({ routes, selectedRouteIds }: Props) => {
   const routeFilterData: RouteFilterData = useRouteFilter()
+  const [isVisible, setIsVisible] = useState(true)
 
   const filteredRoutes = filterRoutes(routes || [], routeFilterData)
 
+  const toggleVisibility = () => setIsVisible(!isVisible)
+
   return (
-    <div className="m-route-picker">
+    <div className={`m-route-picker ${isVisible ? "visible" : "hidden"}`}>
```

Comment:
What do you think about naming these classes `.isVisible` or `.m-route-picker--visible`?

Reply:
`.m-route-picker--visible` would make the selection of `.m-route-picker` much less readable in the css wouldn't it?